### PR TITLE
UIORGS-179 Add translations for permissions

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Approach

https://issues.folio.org/browse/UIORGS-179
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,6 +57,26 @@
...
+          "organizations-storage.categories.item.get"
+        ]
+      },
+      {
+        "permissionName": "ui-organizations.categories.settings.all",
+        "displayName": "Settings (Organizations): Can view and edit categories settings",
+        "visible": true,
+        "subPermissions": [
+          "ui-organizations.categories.settings.view",
+          "configuration.all",
```
Comment (AndreiShumski2) [inline package.json:76]: Is it required?

File: package.json [hunk 2]
```diff
@@ -106,7 +126,7 @@
         "visible": true,
         "subPermissions": [
           "ui-organizations.basic.view",
-          "settings.organizations.enabled"
+          "ui-organizations.categories.settings.view"
```
Comment (AndreiShumski2) [inline package.json:129]: IMO it's not the right approach to have organizations view permissions that include settings. For me `Settings` it's a separate permission set

File: package.json [hunk 3]
```diff
@@ -206,8 +226,7 @@
         "displayName": "Settings (Organizations): Can view and edit settings",
         "visible": true,
         "subPermissions": [
-          "configuration.all",
-          "organizations-storage.categories.all",
+          "ui-organizations.categories.settings.all",
           "ui-organizations.view"
```
Comment (AndreiShumski2) [inline package.json:230]: Looks like it should be removed. With this permission set, the user can get all data from `ui-organizations.basic.view` but from the permission name, it follows that the user can only view and edit Settings (for now it's only categories).

File: package.json [hunk 4]
```diff
@@ -57,6 +57,26 @@
           "settings.enabled"
         ]
       },
+      {
+        "permissionName": "ui-organizations.categories.settings.view",
```
Comment (AndreiShumski2) [inline package.json:61]: You don't need to introduce new permission sets.

File: package.json (hunk 1 above)
Comment (usavkov-epam) [reply package.json]: removed

File: package.json (hunk 2 above)
Comment (usavkov-epam) [reply package.json]: updated

File: package.json (hunk 3 above)
Comment (usavkov-epam) [reply package.json]: removed

File: package.json (hunk 4 above)
Comment (usavkov-epam) [reply package.json]: updated

**Review by NikitaSedyx — APPROVED** [review]

**Review by uladislausamets — APPROVED** [review]

**Review by AndreiShumski2 — APPROVED** [review]

**Pull request merged**